bindings/stronghold-nodejs/src/stronghold.rs

```diff
@@ -236,38 +234,22 @@ impl NapiStronghold {
     Ok(data.into_iter().map(u32::from).collect())
   }
 
-  /// Returns the chain state of the identity specified by `did`.
+  /// Returns the value stored by the identity specified by `did`.
```

Comment:
Nit: doc-comments should be consistent.

```suggestion
  /// Returns the blob stored by the identity specified by `did`.
```